### PR TITLE
Update ops_version.md

### PR DESCRIPTION
**Pull request by kuri-leo**
fix wrong `\n` issue while reviewing Chinese translation and a bug in sample code.

### CODE REVIEW

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by gbaned** [comment]
@miaout17 Can you please take a look on this PR? Thanks!

**Comment by gbaned** [comment]
@kuri-leo Can you please resolve conflicts? Thanks!

**Comment by kuri-leo** [comment]
Thanks for still remember me,

1. Conflict resolved where https://github.com/tensorflow/tensorflow/commit/5b68afa3abfa6fe985068e52be1ff1eb9e57efc3 brings in.
2. Reset original 3 commits https://github.com/tensorflow/tensorflow/commit/3f0b5e9048dca5acef1589efbeb11c1d56f4796e as almost a year, too many commits to merge and rebase with the latest master.
3. Create https://github.com/tensorflow/tensorflow/commit/8348484f45d506471da29b2b08192ec22c3d69f5 to fix the left issue.

Besides, could you please tell what happened? I think this is not an issue that should take so long time to review.

File: tensorflow/lite/g3doc/guide/ops_version.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ is supported for every node in Delegation code.
 ```
 const int kMinVersion = 1;
 TfLiteNode* node;
-TfLiteRegistration;
+TfLiteRegistration registration = nullptr;
 context->GetNodeAndRegistration(context, node_index, &node, &registration);
```
Comment (jdduke) [inline tensorflow/lite/g3doc/guide/ops_version.md]: I think the right fix would be to make sure the result is `kTfLiteOk` here, before proceeding?

Comment (kuri-leo) [reply tensorflow/lite/g3doc/guide/ops_version.md]: You are right, I've added `TF_LITE_ENSURE_STATUS` to ensure status is `kTfLiteOk`. Please review again, thanks! :-)

**Review by jdduke — APPROVED** [review]

**Pull request merged**